app/API/V1/Controllers/UserController.php

```diff
@@ -40,43 +51,80 @@ public function getTTConfig(): array
 
     /**
      * @param SimpleEventContract $event
+     * @throws \Doctrine\DBAL\ConnectionException
      */
-    public function onPostStore (SimpleEventContract $event):void
+    public function onPostStore(SimpleEventContract $event):void
     {
-        if ($event->getEventArgs()['frontEndOptions']['email'] === true) {
-            $result = $event->getEventArgs()['result'];
-            $query = $event->getEventArgs()['params'];
+        $result = $event->getEventArgs()['result'];
 
-            $em = app('em');
+        /** @var UserRepository $userRepo **/
+        $userRepo = $this->getRepo();
+        /** @var User $user **/
+        $user = $userRepo->findOneBy(['id'=>$result['id']]);
 
-            /** @var UserRepository $userRepo */
-            $userRepo = $this->getRepo();
-            $user = $userRepo->findOneBy(['id'=>$result['id']]);
+        /** Set user's default role **/
+        $this->roleRepo->setUserPermissions($user);
```

Comment:
They shouldn't get user level permission until they verify their code.

Reply:
All we should be needing to do in the post persist is sending the email with the verification code in it.

When we screen share I will show you a more scribe way of making the email verification code so it can stay as part of the restful call.

I want to keep everything in the skeleton as scribe driven as possible to use it as an example for other users.

Reply:
Note to self: this can be chained create with a mutator to create the code.